addons/website_event/views/event_templates.xml

```diff
@@ -523,8 +523,10 @@
 <!-- Event - Registration -->
 <template id="registration_template">
     <t t-set="tickets_available" t-value="event.seats_available or event.seats_availability == 'unlimited'"/>
-    <t t-set="buy" t-value="tickets_available and event.state == 'confirm'"/>
-    <form t-if="buy" id="registration_form" t-attf-action="/event/#{slug(event)}/registration/new" method="post" itemscope="itemscope" itemprop="offers" itemtype="http://schema.org/AggregateOffer" class="mb-5">
+    <form t-if="tickets_available
```

Comment:
This code added in second commit (state removal) is modified afterwards. It makes your incremental work difficult to review. Please lessen code rewriting between commits.